src/index.ts

```
@@ -195,7 +197,23 @@ const app = new Elysia({
     if (error instanceof Error) {
       errorMessage = error.message;
     } else if (error != null) {
-      errorMessage = String(error);
+      try {
+        errorMessage = JSON.stringify(error, null, 2);
+      } catch {
+        const errorObj =
+          typeof error === "object"
+            ? Object.getOwnPropertyNames(error).reduce(
+                (acc, key) => ({
+                  ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)